projects/1.18/assets/k-turrets/k-turrets/lang/zh_cn.json

```
@@ -0,0 +1,67 @@
+{
+  "itemGroup.k_turrets": "K-Turrets",
+
+  "item.k_turrets.arrow_turret_item": "箭矢炮塔",
+  "item.k_turrets.bullet_turret_item": "火力炮塔",
+  "item.k_turrets.brick_turret_item": "红砖炮塔",
+  "item.k_turrets.firecharge_turret_item": "火焰弹炮塔",
+  "item.k_turrets.steel_ingot": "钢锭",
+  "item.k_turrets.semi_steel": "半成品钢锭",
+  "item.k_turrets.gauss_bullet": "高斯炮弹",
+  "item.k_turrets.gauss_turret_item": "高斯炮塔",
+  "item.k_turrets.cobble_turret_item": "圆石炮塔",
+
+  "item.k_turrets.brick_drone_item": "红砖无人机",
+  "item.k_turrets.bullet_drone_item": "火力无人机",
+  "item.k_turrets.cobble_drone_item": "圆石无人机",
+  "item.k_turrets.arrow_drone_item": "箭矢无人机",
+  "item.k_turrets.gauss_drone_item": "高斯无人机",
+  "item.k_turrets.firecharge_drone_item": "火球无人机",
+
+  "k_turrets.targets": "目标:",
+  "k_turrets.added": "添加到目标",
+  "k_turrets.add.entity.type": "将新的实体添加到目标列表",
+  "k_turrets.choose.tooltip": "按相应按钮选择要删除的实体",
+  "k_turrets.hold.alt.to.scroll": "要滚动目标列表，请按住Alt并使用鼠标滚轮",
+  "k_turrets.dismantle": "拆除",
+  "k_turrets.clear.list": "清除目标列表",
+  "k_turrets.inventory.hint": "潜行并右键单击炮塔/无人机以查看其库存",
```

```suggestion
  "k_turrets.inventory.hint": "潜行右击炮塔/无人机以查看其库存",
```
这个inventory就有点难顶
和其他类似叫`内容`就有点不大对劲（）